<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomGameWidget</class>
 <widget class="QWidget" name="CustomGameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="GameButton_0_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_0_1">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>10</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_0_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_1_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_1_1">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>70</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_1_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_2_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>135</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_2_1">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>135</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GameButton_2_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>135</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
